kolibri/plugins/facility/assets/src/modules/facilityConfig/actions.js

```diff
@@ -25,23 +23,16 @@ export function saveFacilityName(store, payload) {
 }
 
 export function saveFacilityConfig(store) {
-  store.commit('CONFIG_PAGE_NOTIFY', null);
   const { facilityDatasetId, settings } = store.state;
   const resourceRequests = [
     FacilityDatasetResource.saveModel({
       id: facilityDatasetId,
       data: settings,
     }),
   ];
-  return Promise.all(resourceRequests)
-    .then(function onSuccess() {
-      store.commit('CONFIG_PAGE_NOTIFY', notificationTypes.SAVE_SUCCESS);
-      store.commit('CONFIG_PAGE_COPY_SETTINGS');
-    })
-    .catch(function onFailure() {
-      store.commit('CONFIG_PAGE_NOTIFY', notificationTypes.SAVE_FAILURE);
-      store.commit('CONFIG_PAGE_UNDO_SETTINGS_CHANGE');
-    });
+  return Promise.all(resourceRequests).then(function onSuccess() {
```

Comment:
The test failure [here](https://travis-ci.org/github/learningequality/kolibri/jobs/731420697#L7768) might be because you moved this logic to `FacilityConfigPage`. Do you think you'd be able to update the test (`facilityConfig.spec.js`)

Reply:
The reason the test fails, is because the Promise throws an exception, when it fails and we can't catch it, because the component handles the exception. But if we catch the exception here, then we won't be able to call the snackbar in the component.